2022-04-15/schema.json

```diff
@@ -0,0 +1,214 @@
+                "description": "Date before which the dataset has no data (YYYY-MM-DD)",
+                "type": "string",
+                "pattern": "^\\d{1,4}(-\\d{2}(-\\d{2})?)?$"
+            },
+            "notAfter": {
+                "description": "Date after which the dataset has no data (YYYY-MM-DD)",
+                "type": "string",
+                "pattern": "^\\d{1,4}(-\\d{2}(-\\d{2})?)?$"
+            }
+        },
+        "required": ["notBefore", "notAfter"]
+    },
+    "format": {
+        "description": "Format of the ground truth",
+        "type": "string",
+        "enum": ["Alto-XML", "Page-XML"]
+    },
+    "hands": {
+        "description": "Details about the number of fonts or hands in the dataset",
+        "type": "object",
+        "properties": {
+            "count": {
+                "description": "Number of hands/fonts present in the dataset",
+                "type": "string",
+                "enum": ["1", "1-per-file", "1-per-folder", "less-than-11", "more-than-10", "unknown"]
+            },
+            "precision": {
+                "description": "Level of precision of the count property",
+                "type": "string",
+                "enum": ["exact", "estimated"]
+            }
+        },
+        "required": ["count", "precision"]
+    },
+    "volume": {
+        "description": "Details about the quantity of data",
+        "type": "array",
+        "items": {
+            "type": "object",
+            "properties": {
+                "count": {
+                    "type": "integer",
+                    "minimum": 1
+                },
+                "metric": {
+                    "type": "string",
+                    "enum": ["lines", "regions", "characters", "pages", "images", "files"]
+                }
+            },
+            "required": ["count", "metric"]
+        },
+        "uniqueItems": true,
+        "minItems": 1
+    },
+    "authors": {
+        "description": "Contributors to the project",
+        "type": "array",
+        "items": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the contributor",
+                    "type": "string"
+                },
+                "surname": {
+                    "description": "Surname of the contributor",
+                    "type": "string"
+                },
+                "roles": {
+                    "description": "Roles the contributor had",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "transcriber", "aligner", "project-manager", 
+                            "support", "digitization", "quality-control"
+                        ]
+                    },
+                    "uniqueItems": true
+                }
+            }
+        }
+    },
+    "production-software": {
```

Comment:
Should it be required ? I believe so...

Reply:
Then shouldn't we make `title` required too?

Reply:
Nevermind, I hadn't seen the last item in the schema 😶‍

Reply:
Done!